test/lib/faraday_bang/bang_test.rb

```diff
@@ -8,7 +8,14 @@
 
   [Faraday, Faraday.new].each do |klass|
 
-    Faraday::Connection::METHODS.each do |verb|
+    Faraday::Bang::SUPPORTED_HTTP_METHODS.each do |verb|
+
+      it "##{verb}! passes a block" do
+        block = Proc.new do |req|
+          raise "dummy error #{verb}"
+        end
+        ->{ klass.send("#{verb}!", &block) }.must_raise RuntimeError
```

Comment:
You should probably check to make sure that you're getting the correct error message back. Otherwise this might be a false positive as something else might be raising a `RuntimeError`.